docs/latest/user_docs.rst

```diff
@@ -17,6 +17,7 @@ Learn how to deploy, use, and operate Envoy Gateway.
   user/secure-gateways
   user/tls-cert-manager
   user/tls-passthrough
```

Comment:
good catch, can you also update https://github.com/envoyproxy/gateway/blob/main/docs/v0.4.0/user_docs.rst

Reply:
if its relevant to this release (0.4.0), would be good to add it ^ as well

Reply:
As `0.4.0` version don't have that [file](https://github.com/envoyproxy/gateway/tree/main/docs/v0.4.0/user). So I think there is no need to add it to `0.4.0` version.